rbac/management/group/view.py

```diff
@@ -451,7 +451,13 @@ def principals(self, request, uuid=None):
             else:
                 username_list = []
             proxy = PrincipalProxy()
-            resp = proxy.request_filtered_principals(username_list, account)
+            all_valid_fields = VALID_PRINCIPAL_ORDER_FIELDS + ['-' + field for field in VALID_PRINCIPAL_ORDER_FIELDS]
+            if request.query_params.get(ORDERING_PARAM):
+                sort_field = validate_and_get_key(request.query_params, ORDERING_PARAM, all_valid_fields, 'username')
+                sort_order = 'desc' if sort_field == '-username' else 'asc'
```

Comment:
I do believe the valid value here is `des` for the BOP? https://backoffice-proxy-insights-services.ext.us-east.aws.preprod.paas.redhat.com/docs/#/
```suggestion
                sort_order = 'des' if sort_field == '-username' else 'asc'
```

I know @astrozzc ran into this in #248 as well.